scripts/generic-ci-tests.sh

```diff
@@ -146,7 +146,7 @@ case "$TEST_SUITE" in
 
     "lms-unit")
         case "$SHARD" in
-            "all"|[1-4]|"noshard")
+            "all"|[1-9]|10|"noshard")
```

Comment:
I'm not sure I understand the importance of this line. I guess we prevent invalid shard numbers from running this script? But then why do we not do the same shard checking for cms-unit or commonlib-unit? I feel like we should remove this to avoid confusion... unless of course I'm misunderstanding something.

Reply:
That's totally valid. Will change